proofs.go

```diff
@@ -57,9 +57,63 @@ func VerifySeal(info proof.SealVerifyInfo) (bool, error) {
 	return resp.IsValid, nil
 }
 
+func VerifyAggregateSeals(aggregate proof5.AggregateSealVerifyProofAndInfos) (bool, error) {
+	if len(aggregate.Infos) == 0 {
+		return false, xerrors.New("no seal verify infos")
+	}
+
+	spt := aggregate.Infos[0].SealProof // todo assuming this needs to be the same for all sectors, potentially makes sense to put in AggregateSealVerifyProofAndInfos
```

Comment:
FYI @ZenGround0 

@cryptonemo do you know if this assumption is true?

Reply:
The assumption is true, but it's not technically constrained.  I believe we can aggregate different proof types, but I think for now we are just choosing not to do that.  The FIP should be updated that they should all be the same though since that's the intention for now.